openstack/path_role.go

```diff
@@ -143,7 +143,7 @@ type roleEntry struct {
 	Cloud       string            `json:"cloud"`
 	Root        bool              `json:"root"`
 	TTL         time.Duration     `json:"ttl,omitempty"`
-	SecretType  secretType        `json:"secret_type"`
+	SecretType  secretType        `json:"secret_type,omitempty"`
```

Comment:
Secret type can be either `password` or `token`. Why can it be empty in the storage?